packages/footer/src/Footer.stories.tsx

```diff
@@ -96,13 +96,13 @@ const AgComplexFooter: ComponentStory<typeof Footer> = (args) => {
 			</Flex>
 			<Divider />
 			<Text as="p">Footer text</Text>
-			<Divider />
 			<img
 				className="au-responsive-media-img"
 				src="https://designsystem.gov.au/assets/img/placeholder/157X80.png"
```

Comment:
Please don't link to external resources. That site will likely be decommissioned at some point and this will break.

Reply:
Maybe use the SVG logo from ag-branding